feeds/wlan-ap/opensync/files/bin/lan-ssh-firewall

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+if [ "$#" -ne 1 ]; then
+    echo "Illegal number of parameters"
+    exit 1
+fi
+
+action=$1
+
+index=$(uci show firewall | grep "Lan SSH Rule" | cut -c 16-17)
+
+[ "$action" = "enable" ] && uci set firewall.@rule["$index"].target='REJECT'
```

Comment:
Make REJECT depend on some "developer" flag file in /etc so that we can work easier locally.

Reply:
Added a check for a `/etc/ssh-flag` file before we set the rule to `REJECT`